components/filter/test/filter.test.js

```diff
@@ -311,7 +311,7 @@ describe('d2l-filter', () => {
 
 		it('clear all - clears all dimensions and searches', async() => {
 			const elem = await fixture(`<d2l-filter>
-				<d2l-filter-dimension-set key="1" text="Dim 1" search-type="manual"><d2l-filter-dimension-set-value key="test" text="test" selected></d2l-filter-dimension-set-value></d2l-filter-dimension-set>
+				<d2l-filter-dimension-set key="1" text="Dim 1"><d2l-filter-dimension-set-value key="test" text="test" selected></d2l-filter-dimension-set-value></d2l-filter-dimension-set>
```

Comment:
I'm removing it to simplify the test, yeah. Because now in manual search it will dispatch a search event that would need to be handled. I can clean up that `searchKeysToDisplay` part which looks like it should've been `elem._dimensions[0].searchKeysToDisplay`.

I added a test below that is testing the event when search is set to the empty string, but I think I should add an additional one for `clearAll` and can keep this one testing the automatic search.